packages/notification/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hyperjumptech/monika-notification",
-  "version": "1.15.11",
+  "version": "1.16.0",
```

Comment:
actually no need to update it anymore on every monika release.
according to [this](https://github.com/hyperjumptech/monika/tree/main/packages/notification#releasing), monika/notification should be updated individually every PR related to `packages/notification`
```suggestion
  "version": "1.15.11",
```

Reply:
instead, we need to update the version in `package.json` file line 66
```
"@hyperjumptech/monika-notification": "1.15.7", // update to latest version 1.15.11
```

Reply:
Thank you for the information. The latest version of `@hyperjumptech/monika-notification` is `1.15.7`, but there are some code changes after that release. I think it would be an operational burden to separate the releases. IMO, it would be easier if we just do one release and use one version. But, we should change the release process to release the package in the workspace first, then release the root package. 

## NPM
https://www.npmjs.com/package/@hyperjumptech/monika-notification?activeTab=versions

![image](https://github.com/hyperjumptech/monika/assets/15191978/0c74d908-c3ac-4bea-8f70-980b6be63566)

## GitHub
https://github.com/hyperjumptech/monika/commits/main/packages/notification

![image](https://github.com/hyperjumptech/monika/assets/15191978/571ac54e-f64d-40df-86aa-db872b4095b5)

Reply:
just need to update monika-notification version in root package.json, then resolve this comment

Reply:
Ok, updated. Thank you for the discussion.